### DISCUSSION TITLE
priceLine is changed price value when I zoom in, zoom out or scroll to loadmore

### DISCUSSION THREAD

**Discussion by Linh2196** [klinecharts/KLineChart#277] [Q&A]
I draw a priceLine and zoom, or scroll, it's price value is changed.
I create shape priceLine with fixed pricevalue, it is also changed when I do same

**Comment by liihuu** [comment]
V8.6.0 has fixed  it.

**Reply by ct152098** [reply]
有没有办法创建两条价格线，不是覆盖物那种价格线  覆盖物的价格线在通过鼠标左右拖动时会被y轴覆盖 跑出y轴之外，能不能加一个类似priceMark.last.line 最后一条数据的收盘价的那种线和标签呢？
